core-charttemplatespostcreatehookswds.yaml

```diff
@@ -0,0 +1,695 @@
+
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: transport-controller
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          name: transport-controller
+      template:
+        metadata:
+          labels:
+            name: transport-controller
+        spec:
+          initContainers:
+          - name: setup-wds-kubeconfig
+            image: quay.io/kubestellar/kubectl:{{.Values.KUBECTL_VERSION}}
+            imagePullPolicy: IfNotPresent
+            command: [ "bin/sh", "-c", "sh /mnt/config/get-kubeconfig.sh '{{"{{.ControlPlaneName}}"}}' false | base64 -d > /mnt/shared/wds-kubeconfig" ]
+            volumeMounts:
+            - name: config-volume
+              mountPath: /mnt/config
+            - name: shared-volume
+              mountPath: /mnt/shared
+          - name: setup-its-kubeconfig
+            image: quay.io/kubestellar/kubectl:{{.Values.KUBECTL_VERSION}}
+            imagePullPolicy: IfNotPresent
+            command: [ "bin/sh", "-c", "sh /mnt/config/get-kubeconfig.sh '{{.ITSName}}' true | base64 -d > /mnt/shared/transport-kubeconfig" ]
+            volumeMounts:
+            - name: config-volume
+              mountPath: /mnt/config
+            - name: shared-volume
+              mountPath: /mnt/shared
+          containers:
+          - name: transport-controller
+            image: ghcr.io/kubestellar/kubestellar/ocm-transport-controller:{{.Values.TRANSPORT_VERSION}}
+            imagePullPolicy: IfNotPresent
+            args:
+            - --metrics-bind-address={{.Values.transport_controller.metrics_bind_addr}}
+            - --pprof-bind-address={{.Values.transport_controller.pprof_bind_addr}}
+            - --transport-kubeconfig=/mnt/shared/transport-kubeconfig
+            - --transport-qps={{.Values.transport_controller.transport_qps}}
+            - --transport-burst={{.Values.transport_controller.transport_burst}}
+            - --wds-kubeconfig=/mnt/shared/wds-kubeconfig
+            - --wds-name={{"{{.ControlPlaneName}}"}}
+            - --wds-qps={{.Values.transport_controller.wds_qps}}
+            - --wds-burst={{.Values.transport_controller.wds_burst}}
+            - -v={{.Values.verbosity.transport | default .Values.verbosity.default | default 4 }}
+            - --max-num-wrapped={{.Values.transport_controller.max_num_wrapped}}
+            - --max-size-wrapped={{.Values.transport_controller.max_size_wrapped}}
+            volumeMounts:
+            - name: shared-volume
+              mountPath: /mnt/shared
+              readOnly: true
+          volumes:
+          - name: shared-volume
+            emptyDir: {}
+          - name: config-volume
+            configMap:
+              name: transport-controller-config
+              defaultMode: 0744
+
+  # ^^^^ TRANSPORT PLUGIN ^^^^
+
+  # vvvv KUBESTELLAR PLUGIN vvvv
+
+  - apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
GitHub does not make this clear, but: the lines above this one are terminated with CRLF, while the lines in this section are terminated by LF. This is a bit of a pain; it would be better if all the lines were terminated by LF.

Reply:
I pushed an update forcing the file to LF but I'm confused because my git config should be setup to convert all CRLF to LF

Reply:
Thanks for the file fix.